packages/face-mesh-detection/android/src/main/java/io/capawesome/capacitorjs/plugins/mlkit/facemeshdetection/classes/ProcessImageResult.java

```diff
@@ -0,0 +1,122 @@
+package io.capawesome.capacitorjs.plugins.mlkit.facemeshdetection.classes;
+
+import android.graphics.Rect;
+import com.getcapacitor.JSArray;
+import com.getcapacitor.JSObject;
+import com.google.mlkit.vision.common.PointF3D;
+import com.google.mlkit.vision.common.Triangle;
+import com.google.mlkit.vision.facemesh.FaceMesh;
+import com.google.mlkit.vision.facemesh.FaceMeshPoint;
+import java.util.List;
+
+public class ProcessImageResult {
+
+    private final List<FaceMesh> faceMeshs;
+
+    public ProcessImageResult(List<FaceMesh> faceMeshs) {
+        this.faceMeshs = faceMeshs;
+    }
+
+    public JSObject toJSObject() {
+        JSArray faceMeshsResult = this.createFaceMeshsResult();
+
+        JSObject result = new JSObject();
+        result.put("faceMeshs", faceMeshsResult);
+        return result;
+    }
+
+    private JSArray createFaceMeshsResult() {
+        JSArray result = new JSArray();
+        for (FaceMesh faceMesh : faceMeshs) {
+            JSObject faceMeshResult = this.createFaceMeshResult(faceMesh);
+            result.put(faceMeshResult);
+        }
+        return result;
+    }
+
+    private JSObject createFaceMeshResult(FaceMesh faceMesh) {
+        JSObject result = new JSObject();
+
+        Rect boundingBox = faceMesh.getBoundingBox();
+        JSObject boundsResult = this.createBoundsResult(boundingBox);
+        result.put("bounds", boundsResult);
+
+        JSArray faceMeshPointsResult = new JSArray();
+        List<FaceMeshPoint> faceMeshPoints = faceMesh.getAllPoints();
+        for (FaceMeshPoint faceMeshPoint : faceMeshPoints) {
+            int index = faceMeshPoint.getIndex();
+            PointF3D position = faceMeshPoint.getPosition();
+
+            JSObject faceMeshPointResult = this.createFaceMeshPointResult(index, position);
+            faceMeshPointsResult.put(faceMeshPointResult);
+        }
+        if (faceMeshPointsResult.length() > 0) {
+            result.put("faceMeshPoints", faceMeshPointsResult);
+        }
+
+        //        JSArray contoursResult = new JSArray();
```

Comment:
Please remove the comments.

Reply:
That's actually a point I would like to discuss. There is an API to get the face mesh points by contour type. So you can group the fash meshs that belong to a certain region of the face. However, I think that API has been implemented poorly and since it is a beta, it might change again. But my question is if I should add another field in the response where the face mesh points are grouped by the contour type? WDYT?

Reply:
Oh okay, I didn't take a closer look at it. Sure, we can add that, as long as the app doesn't crash because the payload is too big. ^^

Reply:
Yes, you are right. Let me test it out...